src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.targets

```diff
@@ -7,4 +7,8 @@
     <AdditionalFiles Include="$(MSBuildProjectDirectory)\*.analyzerdata" />
```

Comment:
Thanks for pointing that out! I have a different PR #595 that also addresses that!